project/Build.scala

```diff
@@ -18,7 +18,7 @@ object Scrooge extends Build {
 
   // To build the develop branch you need to publish util, and finagle locally:
   // 'git checkout develop; sbt publishLocal' to publish SNAPSHOT versions of these projects.
-  val utilVersion = "6.34.0" + suffix
+  val utilVersion = "6.35.0" + suffix
```

Comment:
not sure how you handle this internally, my thoughts were that since it depends on the 2.12 branch that has not been released it would be: increment latest release version and add suffix. if i don't increment i am essentially publishing a local snapshot version that already has a corresponding release (6.34.0).